pkg/utils/fs.go

```diff
@@ -38,23 +38,30 @@ import (
 )
 
 // DirSize returns the accumulated size of all files in folder. Result in bytes
-func DirSize(fs v1.FS, path string) (int64, error) {
+func DirSize(fs v1.FS, path string, excludes ...string) (int64, error) {
```

Comment:
We definitely need a unit test exercising this change. I can be easily coded in `pkg/utils/utils_test.go` there is already a context there for this function, so just adding one or two runs to exercise the extra `return` and eventual corner cases should be enough to be sure this is functional and raise on regressions.

Reply:
I am also wondering if `filepath.SkipDir` is equivalent to `fs.SkipDir` from `io/fs` package. 

The function we are using https://github.com/twpayne/go-vfs/blob/master/walk.go seams to honor the `io/fs` version, however I have not had a closer look to check if there is some sort of compatibility or there is some detail I am missing.